gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSObservabilityEventV0.avsc

```diff
@@ -0,0 +1,162 @@
+{
+  "type": "record",
+  "name": "GaaSObservabilityEventV0",
```

Comment:
good wisdom for us to iterate after achieving e2e functionality, so I suggest a top-level "doc" string to clarify the clear intent to supersede this schema.

an alt. suffix that just occurred to me is `Experimental`